src/components/pages/blog/posts.tsx

```diff
@@ -20,7 +20,7 @@ interface PostsProps {
 }
 
 export const Posts = ({ posts }: PostsProps) => {
-  const dateFormatter = useLocaleFormat('dd. MMMM yyyy');
+  const dateFormatter = useLocaleDateFormatter();
```

Comment:
I see you put that into the helper so you don't have to repeat the formatting? I think the name `useLocaleDateFormatter` is not the best as it doesn't give me any ideas how it's formatted.

Something like that would tell a better story ?
```
useLocaleFormat(LONG_DATE');
```